pkgs/tools/security/rnp/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, botan2
+, bzip2
+, cmake
+, fetchFromGitHub
+, fetchurl
+, json_c
+, zlib
+}:
+let
+  versionCmake = fetchurl {
+    url = "https://raw.githubusercontent.com/riboseinc/cmake-versioning/c78a0be/version.cmake";
+    sha256 = "09ghbxyf6kr75857a3v9zql5j1zxnw1fkbndwsyrc1k9qmg24824";
+  };
+  rev = "ffcfb6319ffd5dfe0146bc119a3c8eb994c132a2";
+in
+stdenv.mkDerivation rec {
+  pname = "rnp";
+  version = "v0.14.0-1-g${builtins.substring 0 7 rev}+${toString SOURCE_DATE_EPOCH}";
```

Comment:
Please do not prefix the version with v and adding 1970 to the version is also not necessary.

Reply:
Please see below.